crypto/fipsmodule/rand/cpu_jitter_test.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#if defined(BORINGSSL_FIPS)
+
+#include <gtest/gtest.h>
+
+#include "../../test/test_util.h"
+#include "../../../third_party/jitterentropy/jitterentropy.h"
+
+// Struct for Jitter entropy collector instace with constructor/desctructor.
+struct JitterEC {
```

Comment:
Can we use https://github.com/awslabs/aws-lc/blob/main/include/openssl/ec.h#L399?

Reply:
I couldn't make that work, got C++ compilation errors that I don't fully understand. Do you have strong preference for using the macro or would the current solution be ok?